parsers/UY.py

```diff
@@ -170,17 +181,17 @@ def fetch_consumption(
     )
     data = data.rename(columns={"Demanda": "consumption"})
 
-    consumption = data[["consumption"]].to_dict(orient="index")
-    all_data_points = []
-    for dt in consumption:
-        data_point = {
-            "zoneKey": "UY",
-            "datetime": dt.to_pydatetime().replace(tzinfo=UY_TZ),
-            "consumption": round(consumption[dt]["consumption"], 3),
-            "source": "pronos.adme.com.uy",
-        }
-        all_data_points += [data_point]
-    return all_data_points
+    consumption = data[["consumption"]]
+
+    for dt, row in consumption.iterrows():
+        consumption_list.append(
+            zoneKey=zone_key,
```

Comment:
Same here so we don't end up with type errors.

Reply:
Good catch